package.json

```diff
@@ -50,6 +51,7 @@
     ]
   },
   "devDependencies": {
+    "@types/react-test-renderer": "^17.0.1",
```

Comment:
Hi @DnsZhou  I can see `@types/react-test-renderer` dependency in devDependencies and `react-test-renderer` dependency in dependencies section, it there some special reason?

Reply:
this was recommended by the npm when I missed @types/react-test-renderer for typescript. seems the one with @types is just for development that could enable type script to link the types.